client/coral-embed-stream/src/graphql/index.js

```diff
@@ -200,7 +200,11 @@ const extension = {
       variables: {id, edit},
     }) => ({
       updateQueries: {
-        EmbedQuery: (previousData, {mutationResult: {data: {editComment: {comment: {status}}}}}) => {
+        EmbedQuery: (previousData, {mutationResult: {data: {editComment: {comment, errors}}}}) => {
+          if (errors && errors.length) {
```

Comment:
I added the todo comment.

I disagree that it's a good idea to remove the error checking *before* you do something that makes it unnecessary. If I did that, it would allow the UI to inaccurately reflect a failed ignoreUser and potentially confuse the end-user. When these lines are made unnecessary, then it makes sense to remove them, but not before.